apps/angular/src/app/features/home/anime-table/anime-table.component.ts

```diff
@@ -0,0 +1,41 @@
+import { ChangeDetectionStrategy, Component, inject } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { MatTableModule } from '@angular/material/table';
+import { Observable } from 'rxjs';
+import { AnimeService } from '@js-camp/angular/core/services/anime.service';
+import { Pagination } from '@js-camp/core/models/pagination';
+import { Anime } from '@js-camp/core/models/anime';
+import { NoEmptyPipe } from '@js-camp/angular/core/pipes/no-empty.pipe';
+
+/** Anime Table Component. */
```

Comment:
There is no need to use capital case for documentation.
```suggestion
/** Anime table component. */
```